Task1/Project/Shapes/Models/Classes/ShapeBase.cs

```diff
@@ -8,8 +11,25 @@ namespace Shapes.Models.Classes
     /// </summary>
     public abstract class ShapeBase : IShape, IFileManager
     {       
+        // FIELDS
+        static IdentifierShape idOfTheShape;
+        static Dictionary<string, Deserialization> dictionaryOfId;    
         // PROPERTIES
-         /// <summary>
+        /// <summary>
+        /// The identifier of the shape.
+        /// </summary>
+        public static IdentifierShape IdOfTheShape

```

Comment:
Bad naming and Documentation, was mentioned above

Reply:
Done.